evennia/utils/tests/test_containers.py

```diff
@@ -4,78 +4,71 @@
 from django.conf import settings
 from django.test import override_settings
 from evennia.utils.utils import class_from_module
+from evennia import DefaultScript
 
-_BASE_SCRIPT_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
+_BASE_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
 
-class GoodScript(_BASE_SCRIPT_TYPECLASS):
-  pass
+class GoodScript(DefaultScript):
+    pass
 
-class BadScript:
```

Comment:
I would suggest to add two tests for `BadScript`. It's probably closer to a real error than `BrokenScript` is. `BrokenScript` is very contrived!!
(I added to make sure it would fail during `GlobalScriptContainers.start()`

Apart from that, I think these are all good tests!!

Reply:
What would it test for?

Reply:
Also: the original issue was specifically for when there's an import error on the script class, so `BrokenScript` might be contrived but it's a good way to make a successful test case for it that doesn't prevent the test module from loading.

Reply:
At first, I thought one for `gcs = GlobalScriptContainer()` ("init") and the other for `gcs.start()`.

1. If I am not mistaken, `init` will succeed, because the class exists. It will place it in `_loaded_data`. This would be testing internal implementation - which it's not a good thing to test. Better not to add this test actually. However, I think that will be the end result.
 
2. If I am not mistaken, `start()` will fail with `AttributeError` because `BadScript` does not have `objects` property. So this test will raise as expected. I think this is expected behavior because a bad test should not let the server start.